webapp/templates/jury/contest.html.twig

```diff
@@ -106,6 +106,32 @@
                     <th>Process balloons</th>
                     <td>{{ contest.processBalloons | printYesNo }}</td>
                 </tr>
+                <tr>
+                    <th>Process awards</th>
+                    <td>{{ contest.processAwards | printYesNo }}</td>
+                </tr>
+                <tr>
+                    <th>Awards</th>
+                    <td>
+                        {% if contest.processAwards %}
+                            <a>{{ contest.goldAwards|default('-') }} Gold Medal</a>
+                            <br>
+                            <a>{{ contest.silverAwards|default('-') }} Silver Medal</a>
+                            <br>
+                            <a>{{ contest.bronzeAwards|default('-') }} Bronze Medal</a>
+                            <br>
+                            {% for category in contest.awardsCategories %}
+                                For all teams from
```

Comment:
I will probably move this outside the loop and only show the category names + links

Reply:
Great.